package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Easy-to-use HTML tables in Vue.js",
   "main": "dist/NovaTable.js",
   "scripts": {
-    "test": "karma start karma.conf.js"
+    "test": "mocha-webpack --webpack-config webpack.config.js --require test/setup.js test/**/*.spec.js"
   },
```

Comment:
So, I'm a little stuck on how to build the distribution files now. Can you add a `scripts` line to do that for us?

Reply:
Sure. Will commit updates in the next day or so